modules/oe_theme_content_consultation/config/install/core.entity_view_display.node.oe_consultation.full.yml

```diff
@@ -200,6 +214,15 @@ content:
     third_party_settings: {  }
     type: datetime_default
     region: content
+  oe_consultation_documents:
```

Comment:
oe_consultation_documents field has to be hidden in the `core.entity_view_display.node.oe_consultation.teaser.yml`.